hack/tests/test_heapster.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+TEST_SOURCE=${BASH_SOURCE}
+TEST_DIR=$(dirname "${BASH_SOURCE}")
+
+function tests.setup {
+  #initial setup required for all test scenarios
+  oc create -f $SOURCE_ROOT/metrics-deployer-setup.yaml &> /dev/null || true
+  oadm policy add-role-to-user edit system:serviceaccount:${TEST_PROJECT}:metrics-deployer
+  oadm policy add-cluster-role-to-user cluster-reader system:serviceaccount:${TEST_PROJECT}:heapster
+}
+
+function tests.teardown {
+  oc delete -f $SOURCE_ROOT/metrics-deployer-setup.yaml &> /dev/null || true
+  #clean up required after the tests have run.
+#  oadm policy remove-role-from-user edit system:serviceaccount:${TEST_PROJECT}:metrics-deployer
+#  oadm policy remove-cluster-role-from-user cluster-reader system:serviceaccount:${TEST_PROJECT}:heapster
+}
+
+function checkDeployer {
+  Info "Checking the deployer."
+
+  DEPLOYER_START=$(date +%s)
```

Comment:
Maybe not as part of this PR but it would be useful to start gathering utility functions into a common place.  For instance, this script would be more readable if it said `DEPLOYER_START=Time.now()`